src/clone/pubkey.cpp

```diff
@@ -178,7 +178,7 @@ bool XOnlyPubKey::VerifySchnorr(const uint256& msg, Span<const unsigned char> si
     assert(sigbytes.size() == 64);
     secp256k1_xonly_pubkey pubkey;
     if (!secp256k1_xonly_pubkey_parse(secp256k1_context_verify, &pubkey, m_keydata.data())) return false;
-    return secp256k1_schnorrsig_verify(secp256k1_context_verify, sigbytes.data(), msg.begin(), &pubkey);
+    return secp256k1_schnorrsig_verify(secp256k1_context_verify, sigbytes.data(), msg.begin(), sizeof(msg), &pubkey);
```

Comment:
We shouldn't be making changes to cloned sources. Apparently consensus is out of sync with libsec latest. This is why we maintain our own versioning for libsecp. I didn't catch this because the consensus dependency for libsecp wasn't updated in the last changes.

Reply:
Looks like our version8 is ahead of libconsensus latest release (22.x). As an aside, libconsensus 22.x makes the transition to c++17, and libbitcoin-consensus would require this as well, cascading through other repos. Let's accept this change to the clone files from libconsensus 0.21.x and freeze our c++11 support at that level, along with libsecp/version8.

Reply:
Okay, then I'll revert to version7 and I'll patch the broken test include there which isn't encountered when building without tests.